apps/rest_api/lib/controllers/public/auth.ex

```diff
@@ -47,7 +47,7 @@ defmodule RestAPI.Controllers.Public.Auth do
   @doc "Logout the authenticated subject session."
   @spec sign_out(conn :: Plug.Conn.t(), params :: map()) :: Plug.Conn.t()
   def sign_out(%{private: %{session: session}} = conn, _params) do
-    session
+    session.jti
```

Comment:
@yashin5 this is what was crashing the logout -.-.
I removed this param unintentionally and as the API tests are mocked i didn't noticed at all xD.